packages/data-models/src/treebank_data_item.js

```diff
@@ -18,7 +18,7 @@ export default class TreebankDataItem {
     this.wordIds = []
     this.sentenceId = null
     this.doc = null
-
+    this.suppressTree = false
```

Comment:
Maybe to add a comment explaining why do we suppress a tree (just to be able to recall it later)?

Reply:
good idea :-) will fix.